app/services/claim_fast_tracking/max_rating_annotator.rb

```diff
@@ -18,11 +18,11 @@ def self.annotate_disabilities(rated_disabilities_response)
 
       ratings = get_ratings(diagnostic_codes)
       if ratings.present?
-        ratings.each do |rating|
-          rated_disability = rated_disabilities_response.rated_disabilities.find do |disability|
-            disability.diagnostic_code == rating['diagnostic_code']
+        rated_disabilities_response.rated_disabilities.each do |rated_disability|
+          max_rating = ratings.find do |rating|
+            rated_disability.diagnostic_code == rating['diagnostic_code']
```

Comment:
This isn't a new complaint about this region of code, but I feel like the nesting of the `find` inside the `each` makes for poor reading. (It also smells inefficient but the data structures are small). Do you want to try the route of building a DC➡️max_rating hash out of the VRO response, and see if that feels any better?

Reply:
+1 that this is a bit of a strange pattern.  I would recommend putting in a helper method that describes what it is doing, or at the very least adding a comment that explains what the code is doing.

Reply:
@SamStuckey Yeah it was. I cleaned it up in a more recent commit!